packages/interactivity/src/store.ts

```diff
@@ -23,7 +12,10 @@ const deepMerge = ( target: any, source: any ) => {
 		for ( const key in source ) {
 			const getter = Object.getOwnPropertyDescriptor( source, key )?.get;
 			if ( typeof getter === 'function' ) {
-				Object.defineProperty( target, key, { get: getter } );
+				Object.defineProperty( target, key, {
+					get: getter,
+					configurable: true,
```

Comment:
This is required in case someone wants to overwrite a getter with a value.

Reply:
The use case is when there are multiple calls to the `store` function for the same namespace.